src/core/src/components/formly.form.ts

```diff
@@ -156,6 +156,7 @@ export class FormlyForm implements DoCheck, OnChanges, OnDestroy {
     this.initializeFormValue(this.form);
     (<FormGroup> this.form).patchValue(model, { onlySelf: true });
     this.trackModelChanges(this.fields);
+    this.fields.forEach((field) => field.model = model);
```

Comment:
OK I have added it here. Is this what you mean

Reply:
it's better to move it inside `resetFieldArray` to update nested field too

Reply:
it will be set if it has `fieldArray`

> Only place need to set is in the else statement?

yep